config/test.exs

```diff
@@ -6,7 +6,10 @@ config :maru, Aprb.Api.Root,
 config :aprb, Aprb.Repo,
   pool: Ecto.Adapters.SQL.Sandbox,
   adapter: Ecto.Adapters.Postgres,
-  username: System.get_env("DB_USER"),
-  password: System.get_env("DB_PASSWORD"),
+  # username: System.get_env("DB_USER"),
+  # password: System.get_env("DB_PASSWORD"),
```

Comment:
Did you mean to leave these commented out lines in here?

Reply:
this is little annoying actually, working on fixing it now, but the only way i can run the test outside of Hokusai is to comment this out 😞 

alternative is to setup user/pass locally for my test database and have these default to that...